.github/workflows/release.yaml

```
@@ -35,7 +35,7 @@ jobs:
         with:
           fetch-depth: 0
       - name: Select charmhub channel
-        uses: canonical/charming-actions/upload-charm@2.1.0-rc
+        uses: canonical/charming-actions/upload-charm@2.0.0-rc
```

```suggestion
        uses: canonical/charming-actions/upload-charm@2.1.1
```

See https://github.com/canonical/charming-actions/releases/tag/2.1.1